### PR TITLE
Dropdown: Add cursor pointer to DropdownStyled

### PR DESCRIPTION
**Pull request by ChengYanJin**
**Component**:
Dropdown, Button

**Description**:
When hover to the dropdown/button it's better to show the pointer so that the user can understand it's clickable.

**Design**:

### CODE REVIEW

File: src/lib/components/button/Button.component.js [hunk 1]
```diff
@@ -97,7 +98,8 @@ export const ButtonStyled = styled.button`
         border-color: ${brandingTheme[props.variant]};
         background-color: ${brandingTheme.background};
         color: ${brandingTheme[props.variant]};
-        
+        cursor:pointer;
```
Comment (carlito-scality) [inline src/lib/components/button/Button.component.js]: IMO, we do not need it here.

File: src/lib/components/button/Button.component.js [hunk 2]
```diff
@@ -179,7 +181,8 @@ ${props => {
         border: none;
         background-color: transparent;
         color: ${props.disabled ? brandLight : brandingTheme[props.variant]};
-        
+        cursor:pointer;
```
Comment (carlito-scality) [inline src/lib/components/button/Button.component.js]: IMO, we do not need it here

**Review by carlito-scality — CHANGES_REQUESTED** [review]

File: src/lib/components/button/Button.component.js (hunk 1 above)
Comment (ChengYanJin) [reply src/lib/components/button/Button.component.js]: You are right, thanks!

**Review by carlito-scality — APPROVED** [review]

**Pull request merged**